### DISCUSSION TITLE
webhooks on mysql [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2023] [Discord Threads]
> 🙋‍♂️ **Clement**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### If the insert/update happen via our apis - we trigger it else not. With pg - we plan to hook up pg-notify later  
#  
> 🙋‍♂️ **Clement**  
#### OK thanks, so clearly no. Guess I am going to make a Cron job to look for updates regularly for now. Thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968782152927936522)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
